lib/node_modules/@stdlib/blas/ext/base/dsapxsumpw/src/main.c

```diff
@@ -0,0 +1,135 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
```

Comment:
```suggestion
* Copyright (c) 2024 The Stdlib Authors.
```

We can go ahead and satisfy the linter here. We're making enough changes to the original file that we can update copyright year.

cc @Planeshifter